src/components/src/AppHeaderLogin/appheaderlogin.main.tsx

```diff
@@ -397,24 +397,11 @@ class AppHeaderLoginMain extends Component<AppHeaderLoginMainProps, AppHeaderLog
                     </Link>
                   </li>
                   )}
-                  <li className="dropdown-item">
-                    {(Config.b2b.enable) ? (
-                      <button className="logout-link" type="button" data-el-label="auth.logout" onClick={() => logoutAccountManagementUser()}>
-                        <span className="icon" />
-                        {intl.get('logout')}
-                      </button>
-                    ) : (
-                      <button className="logout-link" type="button" data-el-label="auth.logout" onClick={() => this.logoutRegisteredUser()}>
-                        <span className="icon" />
-                        {intl.get('logout')}
-                      </button>
-                    )}
-                  </li>
                   {(localStorage.getItem(`${Config.b2b.enable && Config.cortexApi.scope}_b2bCart`)) ? (
                     <li className="dropdown-item shop-for">
                       <div className="shopping-as-link">
                         <span>
-                          {intl.get('shopping-as')}
+                          {intl.get('organization')}
```

Comment:
We had recently changed this to "Shopping As" based on feedback as it aligns nicely with the "Shop For" modal. Was it mentioned anywhere else outside the ticket to make this change to "Organization"? Otherwise let's revert this.

Reply:
@shaunmaharaj I'll revert this